nice_updater.sh

```diff
@@ -13,6 +13,8 @@ preferenceFileFullPath="/Library/Preferences/com.github.ryangball.nice_updater.p
 updateInProgressTitle=$(defaults read "$preferenceFileFullPath" UpdateInProgressTitle)
 updateInProgressMessage=$(defaults read "$preferenceFileFullPath" UpdateInProgressMessage)
 loginAfterUpdatesInProgressMessage=$(defaults read "$preferenceFileFullPath" LoginAfterUpdatesInProgressMessage)
+notificationActionButtonText=$(defaults read "$preferenceFileFullPath" notificationActionButtonText)
```

Comment:
Change the key that is being read to NotificationActionButtonText (uppercase N) to stay consistent with the other keys in the plist. The variable can keep the lowercase n.

Reply:
Makes complete sense. Sorry for not spotting it! :)